dlp/snippets/deid/date_shift.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package deid
+
+// [START dlp_deidentify_date_shift]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	dlp "cloud.google.com/go/dlp/apiv2"
+	dlppb "google.golang.org/genproto/googleapis/privacy/dlp/v2"
+)
+
+// deidentifyDateShift shifts dates found in the input between lowerBoundDays and
+// upperBoundDays.
+func deidentifyDateShift(w io.Writer, project string, lowerBoundDays, upperBoundDays int32, input string) error {
```

Comment:
s/project/projectID/g (in all of the samples).

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#common-identifiers

Reply:
Done